aiidalab_widgets_base/structures.py

```diff
@@ -273,14 +313,24 @@ def _default_structure(self):
 
 class StructureExamplesWidget(ipw.VBox):
     """Class to provide example structures for selection."""
+    manager = Instance(StructureManagerWidget, allow_none=True)
     structure = Instance(Atoms, allow_none=True)
 
-    def __init__(self, examples, **kwargs):
+    def __init__(self, examples, title='', **kwargs):
+        self.title = title
         self.on_structure_selection = lambda structure_ase, name: None
         self._select_structure = ipw.Dropdown(options=self.get_example_structures(examples))
         self._select_structure.observe(self._on_select_structure, names=['value'])
         super().__init__(children=[self._select_structure], **kwargs)
 
+    @observe('manager')
+    def _change_manager(self, value):
+        """Set structure manager trait."""
+        manager = value['new']
+        if manager is None:
+            return
+        dlink((self, 'structure'), (manager, 'input_structure'))
```

Comment:
Second duplication, if kept, this should be refactored into an abstract base class. But I hope that we can actually drop this (see previous comment).